Gems/Multiplayer/Code/Source/AutoGen/Multiplayer.AutoPackets.xml

```diff
@@ -11,17 +11,22 @@
         <Member Type="uint16_t" Name="networkProtocolVersion" Init="0" />
         <Member Type="uint64_t" Name="temporaryUserId" Init="0" />
         <Member Type="Multiplayer::LongNetworkString" Name="ticket" />
+        <Member Type="AZ::HashValue64" Name="multiplayerComponentVersionHash" />
     </Packet>
 
     <Packet Name="Accept" HandshakePacket="true" Desc="Server accept packet">
         <Member Type="Multiplayer::LongNetworkString" Name="map" />
     </Packet>
 
+    <Packet Name="SyncComponentMismatch" HandshakePacket="true" Desc="Sends a vector of multiplayer components with their name and version hashes in order to compare which particular components are mismatched. Note: Marked as a handshake packet so it can be sent back to the client without having received the server acceptance packet.">
```

Comment:
Consider changing the name and description so that its clear this is synching component hashes.

Unclear from desc if this is all the hashes or just the ones which are different.

Reply:
Updated the description to mention that all component information is sent in order to compare. Both the server and client receive this packet from one another. Wondering about the name change though, since I'm more concerned about the event (mismatch) than the under-lying hashes. Maybe "component" is making it confusing; I'm thinking "VersionMismatch" instead of "SyncComponentMismatch" would be a better name. Like a "version mismatch" event just occurred.